### PR TITLE
Refactor arearange/parallel axis: use ordered events

### PR DESCRIPTION
**Pull request by TorsteinHonsi**
Used ordered events (`afterTranslate`) in order to avoid `parallelAxis` logic inside the `arearange` module. 

_Disclaimer_
No TypeScript was harmed during this PR. But some any-casts were fixed.

### CODE REVIEW

File: ts/Series/AreaRange/AreaRangePoint.ts [hunk 1]
```diff
@@ -96,13 +96,13 @@ class AreaRangePoint extends AreaPoint {
...
+    public plotHigh?: number = void 0;
 
-    public plotLow: number = void 0 as any;
+    public plotLow?: number = void 0;
 
-    public plotHighX: number = void 0 as any;
+    public plotHighX?: number = void 0;
 
-    public plotLowX: number = void 0 as any;
+    public plotLowX?: number = void 0;
```
Comment (bre1470) [inline ts/Series/AreaRange/AreaRangePoint.ts:105]: If it is optional we do not have assign undefined.

File: ts/Series/ColumnRange/ColumnRangeSeries.ts [hunk 2]
```diff
@@ -167,57 +168,64 @@ class ColumnRangeSeries extends AreaRangeSeries {
...
-                    ]; // don't inherit from column tooltip position - #3372
+
+            if (isNumber(plotHigh) && isNumber(plotY)) {
+                point.plotHigh = safeBounds(plotHigh);
+                point.plotLow = safeBounds(plotY);
+
+                // adjust shape
+                y = point.plotHigh;
+                height = pick(
+                    (point as any).rectPlotY,
```
Comment (bre1470) [inline ts/Series/ColumnRange/ColumnRangeSeries.ts:184]: as `PolarPoint`?

**Review by bre1470 — CHANGES_REQUESTED** [review]
Looks overall good! 👍 

Maybe something to optimize...

**Review by K009 — APPROVED** [review]
👍

**Review by bre1470 — APPROVED** [review]

**Pull request merged**